gtfs/spec/en/reference.md

```diff
@@ -625,12 +625,12 @@ For a given ordered pair of arriving trip and departing trip, the transfer with
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-|  `from_stop_id` | Foreign ID referencing `stops.stop_id` | **Conditionally Required** | Identifies a stop or station where a connection between routes begins. If this field refers to a station, the transfer rule applies to all its child stops. Refering to a station is forbiden for `transfer_types` 4 and 5. |
-|  `to_stop_id` | Foreign ID referencing `stops.stop_id` | **Conditionally Required** | Identifies a stop or station where a connection between routes ends. If this field refers to a station, the transfer rule applies to all child stops. Refering to a station is forbiden for `transfer_types` 4 and 5. |
+|  `from_stop_id` | Foreign ID referencing `stops.stop_id` | **Conditionally Required** | Identifies a stop (`location_type=0`) or a station (`location_type=1`) where a connection between routes begins. If this field refers to a station, the transfer rule applies to all its child stops. It must refer to a stop if `transfer_type` is `4` or `5`.<br><br>Conditionally Required:<br>- **Required** if `transfer_type` is `1`, `2`, or `3`.<br>- Optional if `transfer_type` is `4` or `5`. |
```

Comment:
In the following statement:

> It must refer to a stop if `transfer_type` is `4` or `5` 

Do you mean "stop" as in "location_type=0 or empty", or "everything except stations i. e. location_type=0, 3 or 4"?

Reply:
> Identifies a stop (`location_type=0`) or a station (`location_type=1`) where a connection between routes begins.

This sentence should already limit to location_type 0 or 1 for this field.

Added "(`location_type=0`)" after the "stop" - e785501